WireAnalytics/Package.swift

```diff
@@ -8,15 +8,20 @@ let package = Package(
     platforms: [.iOS(.v15), .macOS(.v12)],
     products: [
         .library(name: "WireAnalytics", targets: ["WireAnalytics"]),
-        .library(name: "WireDatadog", targets: ["WireDatadog"])
+        .library(name: "WireDatadog", targets: ["WireDatadog"]),
+        .library(name: "WireAnalyticsSupport", targets: ["WireAnalyticsSupport"])
     ],
     dependencies: [
-        .package(url: "https://github.com/DataDog/dd-sdk-ios.git", exact: "2.16.0")
+        .package(url: "https://github.com/DataDog/dd-sdk-ios.git", exact: "2.16.0"),
+        .package(url: "https://github.com/Countly/countly-sdk-ios.git", exact: "24.4.2"),
+        .package(path: "../SourceryPlugin")
     ],
     targets: [
         .target(
             name: "WireAnalytics",
-            dependencies: resolveWireAnalyticsDependencies(),
+            dependencies: resolveWireAnalyticsDependencies() + [
+                .product(name: "Countly", package: "countly-sdk-ios")
```

Comment:
I would have expect the same setup as WireDatadog, WireCountly target that is conditionally include if ENABLE_COUNTLY was in. That way on C builds this is excluded completely

Reply:
We're going to tackle this next sprint as discussed.